internal/attestation/renderer/chainloop/v02.go

```diff
@@ -194,6 +199,10 @@ func normalizeMaterial(material *slsa_v1.ResourceDescriptor) (*NormalizedMateria
 		return nil, fmt.Errorf("material name not found")
 	}
 
+	if v, ok := material.Annotations[AnnotationMaterialCAS]; ok && v.(bool) {
+		m.DownloadableFromCAS = true
```

Comment:
change to uploaded to CAS

Reply:
Based on our conversation in the morning. Yes, it will be more consistent this way.